app/src/main/kotlin/com/softteco/template/MainActivity.kt

```diff
@@ -31,14 +31,17 @@ class MainActivity : ComponentActivity() {
     @Inject
     lateinit var profileRepository: ProfileRepository
 
+    @Inject
+    lateinit var bluetoothHelper: BluetoothHelper
+
     @Inject
     @Named("themeMode")
     lateinit var dataStore: DataStore<Preferences>
 
     override fun onCreate(savedInstanceState: Bundle?) {
         installSplashScreen()
         super.onCreate(savedInstanceState)
-        BluetoothHelper.initBluetoothHelper(this)
+        bluetoothHelper.init(this)
```

Comment:
If we have init with passing activity as parameter in onCreate, we have to have opposite function in onDestroy for removing activity from the helper class